### DISCUSSION TITLE
Generate gcode only

### DISCUSSION THREAD

**Discussion by G3Dmaker** [meerk40t/meerk40t#2712] [Q&A] [answered]
Is it possible to generate offline (not connected to machine) gcode?
I have a DIY CNC milling machine.
The spindle can be exchanged with an LED laser module. The controller is a RADDS and Marlin FW (CNC/Laser Config).
Thanks for your help.

**Comment by MarcoRitrivi** [comment, marked as answer]
yes just click save in tab Plan, and it will save the .gcode file.
<img width="350" alt="image" src="https://github.com/user-attachments/assets/95d3aca5-3b67-400e-8193-ece8a3ddd3f9" />
